sdk/core/core-lro/review/core-lro.api.md

```diff
@@ -0,0 +1,95 @@
+## API Report File for "@azure/core-lro"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { AbortSignal } from '@azure/abort-controller';
+import { HttpOperationResponse } from '@azure/core-http';
+import { RequestOptionsBase } from '@azure/core-http';
+
+// @public (undocumented)
+export type LongRunningOperationStates = "InProgress" | "Succeeded" | "Failed" | "Canceled" | "Cancelled";
```

Comment:
the too many `canceled` problem again

Reply:
I didn't change this! :) I only changed one file.